src/apiCalls.js

```diff
@@ -0,0 +1,28 @@
+export const fetchAllMovies = () => {
+  return fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+  .then(res => {
+    if (!res.ok) {
+        return `${res.status} error. Sorry! Something went wrong! Try again later or go to Contact Us to contact the developers with questions!`
+      } else {
+        return res.json()
+      }})
+  }
+
+export const fetchSpecificMovieDetails = (id) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+  .then(res => {
+    if (!res.ok) {
+        return `${res.status} error. Sorry! Something went wrong! Try again later or go to Contact Us to contact the developers with questions!`
+      } else {
+        return res.json()
+      }})
+  }
+ export const fetchTrailerKey = (id) => {
+   return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`)
+     .then(res => {
+       if (!res.ok) {
+           return `${res.status} error. Sorry! Something went wrong! Try again later or go to Contact Us to contact the developers with questions!`
+         } else {
+           return res.json()
+         }})
```

Comment:
Looks good, but I'm thinking we can DRY these up by having just ONE fetch with a parameter for the last part of the endpoint:

`return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/{parameter}`

where the argument is either `movies` `movies/id` or `movies/id/videos` ... (the id would obviously be defined specifically wherever these functions are called)

This would work since all the promises are identical.